### ISSUE TITLE
DD Forms: Reduce Whitespace

### ISSUE THREAD

**Issue by PreciseNZ** [DeskDirector/UserVoice#336]
Would be good to reduce the overall white space in forms to allow for more info to be shown in a single page.

**Comment by Nness** [comment]
Form or form editor?

**Comment by Lowell-Syncrety** [comment]
True... there's about an inch of white space between each question in a form (from the user's perspective in a form). That's a very lot.
